src/_macosx.m

```diff
@@ -489,6 +508,10 @@ static CGFloat _get_device_scale(CGContextRef cr)
          (PyCFunction)FigureCanvas_flush_events,
          METH_NOARGS,
          NULL},  // docstring inherited
+        {"set_cursor",
+         (PyCFunction)FigureCanvas_set_cursor,
+         METH_VARARGS,
+         "Sets the active cursor."},
```

Comment:
The previous imperative tense was correct (and the ones below are wrong.)
```suggestion
         "Set the active cursor."},
```

Reply:
:+1: I updated the others while I was at it as well.